bin/configs/kotlin-jvm-spring-3-restclient-echo-api.yaml

```diff
@@ -0,0 +1,9 @@
+generatorName: kotlin
+outputDir: samples/client/echo_api/kotlin-jvm-spring-3-restclient
```

Comment:
can you please also add the new folder to `.github/workflows/samples-kotlin-*` files?

Reply:
Added